tests/Microsoft.Identity.Test.Unit/AppConfigTests/PublicClientApplicationBuilderTests.cs

```diff
@@ -623,78 +623,5 @@ public void AuthorityWithTenant()
 
             Assert.AreEqual($"https://login.microsoftonline.com/{TestConstants.TenantId}/", app6.Authority);
         }
-
-#if NET6_WIN
```

Comment:
If we soft-obsolete these, better to leave the tests?

Reply:
If we do not support them, get rid of the tests also.

Reply:
Hmm, well these tests test that broker is available on certain platforms, so I think they still may be useful?